src/components/Members.tsx

```diff
@@ -28,7 +29,7 @@ interface DAOInferredProps {
 }
 
 type ArcProps = RequiredProps & ArcInferredProps;
-type DAOProps = RequiredProps & DAOInferredProps;
+type DAOProps = RequiredProps & DAOInferredProps & ArcInferredProps;
```

Comment:
This is unnecessary, because we can get the arc config by doing `this.props.dao.context`. Also arcConfig isn't being used in DAOMembers anywhere.